swarm/network/stream/streamer_test.go

```diff
@@ -1178,6 +1181,11 @@ stream registration, then tests that there are subscriptions.
 */
 func TestGetSubscriptionsRPC(t *testing.T) {
 
+	if testutil.RaceEnabled && os.Getenv("TRAVIS") == "true" {
+		t.Skip("flaky with -race on Travis")
```

Comment:
@janos note: this test fails more often on Travis with `-race`

Reply:
Maybe we should check if it would improve with changes from https://github.com/ethersphere/go-ethereum/pull/1244?

Reply:
I would separate the investigation, so we can have the race detector job finally in place. But yes, definitely we could try.

Reply:
tracker: https://github.com/ethersphere/go-ethereum/issues/1248